cl/favorites/templates/user_prayers.html

```diff
@@ -0,0 +1,145 @@
+{% block og_description %}CourtListener lets you request purchase of legal documents. View the document requests for {{ requested_user }}.{% endblock %}
+
+{% block content %}
+  <div class="col-xs-12">
+    <h1 class="text-center">{% if is_page_owner %}Your RECAP Requests {% else %}RECAP Requests for: {{ requested_user }}{% endif %}</h1>
+  </div>
+
+  <div class="col-xs-12" id="prayer_summary" hx-swap-oob="true"
+    {% if request.user.is_authenticated %}
+    hx-get="{% url 'user_prayers' request.user.username %}"
+    hx-trigger="prayersListChanged from:body" hx-swap="none"
+    {%endif %}
+  >
+    <div class="well well-sm">
+      {% if is_page_owner %}
+      <p>
+        You have had <b>{{ count|intcomma }}</b> requests fulfilled for documents that cost <b>${{total_cost|floatformat:2 }}</b>.
+      </p>
+      <p>
+        {% if is_eligible %}You are eligible to make document requests.{% else %}You have reached your daily limit; wait 24 hours to make new requests.{% endif %}
+      </p>
+      {% endif %}
+    </div>
+  </div>
+
+  <div class="col-xs-12" id="prayer_list" hx-swap-oob="true">
+    <div class="table-responsive">
+      <table class="settings-table table">
+        <thead>
+            <tr>
+              <th>Court</th>
+              <th>Case Name</th>
+              <th>Document Number</th>
+              <th>Document Description</th>
+              <th>Requested On</th>
+              <th>Status</th>
+              {% if is_page_owner %}<th>Action</th>{% endif %}
+            </tr>
+        </thead>
+        <tbody>
+            {% for prayer in prayers %}
+            <tr>
+              <td>{{ prayer.docket_entry.docket.court.citation_string }}</td>
+              {% with docket=prayer.docket_entry.docket %}
+              <td>
+                <a href="{% url "view_docket" docket.pk docket.slug %}">
+                  {{ prayer.docket_entry.docket|best_case_name|safe|v_wrapper }} ({{ prayer.docket_entry.docket.docket_number }})
+                </a>
+              </td>
+              <td>
+                <a href="{% url "view_docket" docket.pk docket.slug %}#entry-{{ prayer.docket_entry.entry_number }}">
+                  {{ prayer.document_number }}
+                </a>
+              </td>
+              {% endwith %}
+              <td>{{ prayer.description }}</td>
+              <td>{{ prayer.date_created|date:"M j, Y"  }}</td>
+              <td>{% if prayer.prayer_status == 1 %} Pending {% elif prayer.prayer_status == 2 %}
+                <div class="btn-group hidden-xs col-sm-4 col-md-3 hidden-print flex">
+                  {% if rd.filepath_local %}
+                    <a href="{{ rd.filepath_local.url }}"
+                        rel="nofollow"
+                        role="button"
+                        class="btn btn-primary btn-xs"
+                            {% if rd.date_upload %}
+                        title="Uploaded {{ rd.date_upload|timezone:timezone }}"
```

Comment:
This line raises a `VariableDoesNotExist` error when loading the page with a `GRANTED` document because the `timezone` is not defined.

If we are going to display `date_upload` in UTC, we could use the custom filter: `datetime_in_utc`:

`{{ prayer.date_upload|datetime_in_utc }}`

This issue is also present in `prayer_email.html`

which could be the reason some emails are being missed, as seen in sentry: [COURTLISTENER-8FC](https://freelawproject.sentry.io/issues/6026700048/)

Reply:
Great catch. Thanks!